projs/shadow/shadow-engine/shadow-renderer/src/vulkan/VulkanModule.cpp

```diff
@@ -194,6 +194,8 @@ void VulkanModule::BeginRenderPass(const std::unique_ptr<vlkx::RenderCommand>& c
                             ImGui_ImplSDL2_NewFrame();
                             ImGui::NewFrame();
 
+                            ImGui::ShowDemoWindow();
```

Comment:
We now have two demo windows being shown?

Reply:
I didn't have one when running. This is an example of why I want #13 in before this.